configs/NEUTRONRCF411AIO/config.h

```diff
@@ -26,17 +26,17 @@
 #define BOARD_NAME        NEUTRONRCF411AIO
 #define MANUFACTURER_ID   NERC
 
-#define USE_ACC
-#define USE_ACC_SPI_ICM42688P
-#define USE_ACC_SPI_ICM42605
 #define USE_GYRO
 #define USE_GYRO_SPI_ICM42688P
-#define USE_GYRO_SPI_ICM42605
-#define USE_GYRO_SPI_ICM20689
-#define USE_ACC_SPI_ICM20689
-#define USE_ACC_SPI_MPU6000
 #define USE_GYRO_SPI_MPU6000
+#define USE_ACC
+#define USE_ACCGYRO_BMI270
+#define USE_ACC_SPI_ICM42688P
+#define USE_ACC_SPI_MPU6000
```

Comment:
Where are the gyro IMU defines for these accelerometers types?

Reply:
need:
```
#define USE_GYRO_SPI_ICM42688P
#define USE_GYRO_SPI_MPU6000
```